src/js/media-fullscreen-button.js

```diff
@@ -15,31 +15,28 @@ const exitFullscreenIcon = `
 </svg>
 `;
 
+const api = {
+  enter: "requestFullscreen",
+  exit: "exitFullscreen",
+  event: "fullscreenchange",
+  element: "fullscreenElement",
+  error: "fullscreenerror",
+};
+
+if (!document.fullscreenElement) {
```

Comment:
This probably needs to be `document.fullscreenElement !== undefined`, otherwise it's usually `null` at this point in Chrome.

Reply:
good catch